code/modules/uplink/uplink_item.dm

```diff
@@ -549,7 +549,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "A cyborg designed and programmed for systematic extermination of non-Syndicate personnel."
 	item = /obj/item/weapon/antag_spawner/nuke_ops/borg_tele
 	cost = 80
-
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/gang)
```

Comment:
Umm you put it at the wrong place that's for the Syndicate Cyborg correct line wrong place put it at line 567.